index.html

```diff
@@ -159,15 +159,70 @@ <h3 id="setup">Setup</h3>
             <p>From the base of the keywhiz repository, build the server:</p>
             <pre class="prettyprint">$ mvn package -am -pl server</pre>
 
+            <p>Create your own configuration files:</p>
+            <pre class="prettyprint">$ cp server/src/main/resources/keywhiz-development.yaml server/src/main/resources/keywhiz-local.yaml
+$ sed -i 's/dev_and_test_keystore/local_keystore/' server/src/main/resources/keywhiz-local.yaml
+$ sed -i 's/dev_and_test_truststore/truststore/' server/src/main/resources/keywhiz-local.yaml
+$ sed -i 's/ponies/password/' server/src/main/resources/keywhiz-local.yaml
+            </pre>
+
+            <p>Create local keystore and truststore (replace your-host)</p>
+            <pre class="prettyprint">
+$ cd server/src/main/resources/
+$ keytool -genkey -alias server-alias -keyalg RSA -storetype JCEKS -keypass password -storepass password -keystore local_keystore.jceks
```

Comment:
we were considering switch the storetype to pkcs12 (Java supports it and more tools support that format vs JCEKS), what do you think?